### PR TITLE
Create pull_request_template.md

### PR DESCRIPTION
**Pull request by tabaktoni**

### CODE REVIEW

**Review by ivpavici — COMMENTED** [review]
LGTM

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+## Motivation and Resolution
+...
+
+### RPC version
```
Comment (janek26) [inline .github/pull_request_template.md:4]: should this be mandatory ? The change could also come from something different than an roc version change

Comment (ivpavici) [reply .github/pull_request_template.md:4]: True, could add next to it something like "if applicable"

Comment (ivpavici) [reply .github/pull_request_template.md]: [cd7a8ac](https://github.com/0xs34n/starknet.js/pull/302/commits/cd7a8ac2e049595d0bc78a5c4cd7c3cd0582c8df)

**Review by janek26 — APPROVED** [review]

**Pull request merged**